src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -84,6 +84,10 @@ public CommandResult execute(Model model) throws CommandException {
             // Should not happen since "1" is a valid index, and import file must have at least 1 entry.
             throw new CommandException(e.getMessage());
         }
+
+        List<Person> importedList = model.getFilteredPersonList();
+        model.setViewingPerson(importedList.get(0));
```

Comment:
Maybe this is not needed because a view command is called already in line 82?

Reply:
Ah was blind and did not see. Thanks for catching this :)